ckanext/security/controllers.py

```diff
@@ -111,8 +111,8 @@ def login(self):
                 # Increment the throttle counter if the login failed.
                 throttle.increment()
 
-            if locked_out or invalid_login:
-                log.info('login failed')
+            if invalid_login or (invalid_login and locked_out):
+                log.info('login failed for {}'.format(user_name))
```

Comment:
Does it matter that now if they have a valid login but are locked out, they no longer get a 403 status?

Reply:
Ah, answered #39 huh?